pysmiles/write_smiles.py

```diff
@@ -187,3 +189,38 @@ def keyfunc(idx):
 
     smiles += ')' * branch_depth
     return smiles
+
+def write_smiles(molecule, default_element='*', start=None):
+    """
+    Creates a SMILES string describing `molecule` according to the OpenSMILES
+    standard. If `molecule` consists of multiple disconnected components their
+    corresponding SMILES string will be joined by zero-order bonds (".").
+
+    Parameters
+    ----------
+    molecule : nx.Graph
+        The molecule for which a SMILES string should be generated.
+    default_element : str
+        The element to write if the attribute is missing for a node.
+    start : Hashable
+        The atom at which the depth first traversal of the molecule should
+        start. A sensible one is chosen: preferably a terminal heteroatom.
+
+    Returns
+    -------
+    str
+        The SMILES string describing `molecule`.
+    """
+    smiles = []
+    for nodes in nx.connected_components(molecule):
+        if start and start in nodes:
```

Comment:
I think in the current implementation, passing the index 0 as start would have been an issue, since 'if 0' evaluates to False. Networkx index labels can be both str or int.
```suggestion
        if not start is None and start in nodes:
```